### PR TITLE
Rfairooz/learn

### LINKED ISSUE #99: Setup Bootstrap Grid in `Learn.js`
- Setup React Bootstrap Grid to have 3 columns and each column will have numerous Topic.js components

### PR DESCRIPTION
**Pull request by rfairooz**
The files I changed on my other branch show up here as commits.

### CODE REVIEW

File: pages/events.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import React from "react";
 import Calendar from "../components/Calendar";
 import PastEvents from "../components/PastEvents";
+import Topic from "../components/Topic";
```
Comment (shahdivyank) [inline pages/events.js]: Remove `Topic` from here

File: pages/events.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
 const Events = () => {
 	return (
 		<div>
 			<Calendar />
 			<PastEvents />
+			<Topic />
```
Comment (shahdivyank) [inline pages/events.js]: Here too

File: components/Learn.js [hunk 3]
```diff
@@ -1,7 +1,22 @@
 import React from "react";
+import { Row, Col } from "react-bootstrap";
 
 const Learn = () => {
-	return <div>Learn</div>;
+	return (
+		<div className='font-Arial text-lg'>
```
Comment (shahdivyank) [inline components/Learn.js]: `font-lexend`

File: components/Learn.js [hunk 4]
```diff
@@ -1,7 +1,22 @@
...
+			{" "}
+			<div className='flex justify-center items-center font-Arial font-bold text-4xl mb-4'>
```
Comment (shahdivyank) [inline components/Learn.js]: `font-lexend

File: components/Learn.js [hunk 5]
```diff
@@ -1,7 +1,22 @@
...
+				Learn
+			</div>
+			<Row>
+				<Col className='flex justify-around'>professional</Col>
```
Comment (shahdivyank) [inline components/Learn.js]: Add Topic components in here:


```Javascript

<Col ...>
    <Topic topic = "Resume" color = "cool_color" />
    <Topic topic = "Academics" color = "cool_color" />
</Col>
```

**Review by shahdivyank — CHANGES_REQUESTED** [review]

File: components/Learn.js [hunk 6]
```diff
@@ -1,7 +1,30 @@
...
 const Learn = () => {
-	return <div>Learn</div>;
+	return (
+		<div className='font-lexend text-lg'>
+			{" "}
+			<div className='flex justify-center items-center font-lexend font-bold text-4xl mb-4'>
+				Learn
+			</div>
+			<Row>
+				<Col className='flex justify-around'>
```
Comment (shahdivyank) [inline components/Learn.js]: Apply the same styles for each column, for right now, I would just stick with `flex justify-center items-center`

**Review by shahdivyank — CHANGES_REQUESTED** [review]

**Review by shahdivyank — APPROVED** [review]

**Review by vwong031 — APPROVED** [review]

**Pull request merged**